packages/vuetify/src/components/VMenu/mixins/menu-keyable.js

```diff
@@ -13,6 +13,13 @@ import { keyCodes } from '../../../util/helpers'
 
 /* @vue/component */
 export default {
+  props: {
+    keyable: {
```

Comment:
I know that we have older exceptions else where, but I tend to lean towards props that require boolean `true` to be active as opposed to writing `:prop="false"`. Thoughts?

Reply:
i'm ok with `disableKeys` prop (default: `false`)